npm/pod_test.go

```diff
@@ -0,0 +1,173 @@
+package npm
```

Comment:
You added a bunch of new files - I think we need to add correct copyright info in those.

Reply:
check the top of https://github.com/Azure/azure-container-networking/blob/master/cni/network/network.go . 
Please check with Sushant if these are needed. I think we need to have Microsoft under copyright statement for new files.

Reply:
Thanks. I added Microsoft copyright info.